corehq/apps/api/resources/v0_5.py

```diff
@@ -489,3 +493,70 @@ def dehydrate(self, bundle):
         bundle.data['product_name'] = bundle.obj.sql_product.name
         bundle.data['transaction_date'] = bundle.obj.report.date
         return bundle
+
+
+UserDomain = namedtuple('UserDomain', 'domain_name project_name')
+UserDomain.__new__.__defaults__ = ('', '')
+
+
+class UserDomainsResource(Resource):
+    domain_name = fields.CharField(attribute='domain_name')
+    project_name = fields.CharField(attribute='project_name')
+
+    class Meta:
+        resource_name = 'user_domains'
+        authentication = ApiKeyAuthentication()
+        object_class = UserDomain
+        include_resource_uri = False
+
+    def obj_get_list(self, bundle, **kwargs):
+        return self.get_object_list(bundle.request)
+
+    def get_object_list(self, request):
+        couch_user = CouchUser.from_django_user(request.user)
+        results = []
+        for domain in couch_user.get_domains():
+            if not ZAPIER_INTEGRATION.enabled(domain):
```

Comment:
It's check for each of user domain. I don't understand how it can be moved outside the loop.